docs/actions.md

```diff
@@ -5,28 +5,30 @@ Use actions to update the state.
 ```jsx
 app({
   state: "Hi.",
-  view: (state, actions) => (
-    <h1 onclick={actions.ucase}>{state}</h1>
-  ),
+  view: (state, actions) =>
+    <h1 onclick={actions.ucase}>
+      {state}
+    </h1>,
   actions: {
     ucase: state => state.toUpperCase()
   }
 })
 ```
 
-To be able to update the state, an action must return a new state or a part of it.
+An action must return a new state or a part of it in order to update the state.
```

Comment:
While that is an interesting example, I can see it being a little scary to a newcomer.

The shallow merging behavior of objects, which you described in another document, would IMO best be illustrated by a simpler example like the one I posted before.   Since 99% of the examples are using a basic primitive for state, it may not be immediately clear to people how to update an object.

Either way, I'm liking the new documentation :+1: